app/External/GitHub/OrganizationApi.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\External\GitHub;
+
+use App\External\ApiProgress;
+use GuzzleHttp\Client;
+use GuzzleHttp\RequestOptions;
+use Illuminate\Support\Collection;
+
+class OrganizationApi
+{
+    use GitHubApiTrait;
+
+    private string $accessToken;
+
+    private Client $client;
+
+    private string $organizationName;
+    private string $organizationUrl;
+
+    /**
+     * TeamApi constructor.
+     */
+    public function __construct($name, string $accessToken)
+    {
+        $this->organizationName = $name;
+        $this->accessToken = $accessToken;
+
+        $this->organizationUrl = "https://api.github.com/orgs/$name";
+        $this->client = new Client();
+    }
+
+    public function team($name): TeamApi
+    {
+        return new TeamApi($this->organizationName, $name, $this->accessToken);
+    }
+
+    public function list(ApiProgress $progress = null): Collection
```

Comment:
can we call this `members` or `listMembers`? It would help the readability of the block in the `ClearOut` command.

Reply:
Um. Yes. Yes we can. That's smart actually, why did I just call it list 😆